### DISCUSSION TITLE
Page layout

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#178] [General]
Hello!
Will there be a typical page layout:
- Header
- Content
- Footer
- Side panels
Etc

Or do it yourself or using other CSS frameworks?

**Comment by waterplea** [comment]
Hello. We do not plan on adding those. You can use dedicated layout CSS libraries together with our components.

**Comment by ghost** [comment]
Got it, thanks for the quick answer :-)
